frontend/packages/ceph-storage-plugin/src/components/ocs-install/ocs-install.scss

```diff
@@ -43,3 +43,18 @@
   margin-top: var(--pf-global--spacer--sm);
   margin-bottom: var(--pf-global--spacer--sm);
 }
+
+// Restrict form widths
+.ceph-ocs-install__node-list {
```

Comment:
@a2batic If I am not mistaken, these are the same styles that gets applied with `co-m-pane__form`